package.json

```diff
@@ -8,7 +8,7 @@
     "Adam Rudd <damvrr@gmail.com>"
   ],
   "scripts": {
-    "test": "tape test.js | faucet",
+    "test": "tape test.js | faucet && standard",
     "ci": "tape test.js && node testRandom"
```

Comment:
We will need to add standard also here. We can even move from faucet to tap-spec.